README.md

```diff
@@ -7,6 +7,9 @@ bootstrap Script for migrating existing running systems to Foreman with the Kate
 This script can take a system that is registered to Spacewalk, Satellite 5, Red Hat
 Network Classic and get it registered to Foreman & Katello.
 
+Optionally, you can also move systems from one Satellite 6 system or Capsule to
```

Comment:
I think moving between different Satellite/Katello installs should not be done with `--new-capsule` as that expects the host to be already present in Foreman to be able to edit it.

How about:
```
Optionally, you can also move systems between Capsules (both internal and external) of one Katello installation by using the `--new-capsule` option.
```